src/main/java/core/basesyntax/dao/impl/CommentDaoImpl.java

```diff
@@ -12,21 +16,67 @@ public CommentDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Comment create(Comment entity) {
-        return null;
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.persist(entity);
+            transaction.commit();
+        } catch (HibernateException ex) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new RuntimeException("Can't add comment to DB: "
+                    + entity + ex);
+        } finally {
+            if (session != null) {
+                session.close();
+            }
+        }
+        return entity;
     }
 
     @Override
     public Comment get(Long id) {
-        return null;
+        try (Session session = factory.openSession()) {
+            return session.get(Comment.class, id);
+        } catch (HibernateException ex) {
+            throw new RuntimeException("Can't get comment from DB by id: "
+                    + id + ex);
```

Comment:
```suggestion
            throw new RuntimeException("Can't get comment from DB by id: "
                    + id, ex);
```
This is a huge difference. With passing ex as a second argument you will see the detail stacktrace

Reply:
oOO, I see